packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -1,30 +1,38 @@
-import type { RefObject, MutableRefObject } from 'react'
-import { useEffect, useState, useCallback, useRef, useMemo } from 'react'
+import type { MutableRefObject, RefObject } from 'react'
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react'
 
 import type { ChatMessageWithExtras } from '@audius/common'
 import {
-  useCanSendMessage,
-  chatCanFetchMoreMessages,
-  chatActions,
+  Status,
   accountSelectors,
+  chatActions,
+  chatCanFetchMoreMessages,
   chatSelectors,
-  encodeUrlName,
   decodeHashId,
   encodeHashId,
-  Status,
+  encodeUrlName,
```

Comment:
Most of the changes up here are due to an `Organize Imports` removing unused items and resorting the rest.